shared/haml/course_explorer_table.haml

```diff
@@ -56,7 +56,7 @@
             - unless course[:starts] == 0
               %div{class: "bar-#{course[:starts]}"}
                 &nbsp;
-            %div{class: "courserow bar-#{course[:ends] - course[:starts] + 1}", style: "background-color: #{"#1AC7D6" || course[:color] || '#00adbc'}"}
+            %div{class: "courserow bar-#{course[:ends] - course[:starts] + 1}", style: "background-color: #{"#00adbc"}"}
```

Comment:
presumably we don't need the `#{}` portion now that there's no ruby in there (just a string)